AssertionError: Invalid admin level. Valid values are [0, 1, 2] in `admin osm` command

reproduction steps

```shell
pipenv run python -m cbsurge.cli admin osm -b "27.767944,-5.063586,31.734009,-0.417477" -l 0 
```

error log
```shell
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/workspace/cbsurge/cli.py", line 13, in <module>
    cli()
  File "/workspace/.venv/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.venv/lib/python3.12/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/workspace/.venv/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.venv/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.venv/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.venv/lib/python3.12/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/cbsurge/admin/__init__.py", line 88, in osm
    fetch_osm_admin(bbox=bbox, admin_level=admin_level,osm_level=osm_level, clip=clip, h3id_precision=h3id_precision)
  File "/workspace/cbsurge/admin/osm.py", line 219, in fetch_admin
    assert admin_level in ADMIN_LEVELS, f'Invalid admin level. Valid values are {list(ADMIN_LEVELS.keys())}'
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: Invalid admin level. Valid values are [0, 1, 2]
```

--- a/cbsurge/admin/osm.py
+++ b/cbsurge/admin/osm.py
@@ -213,6 +213,10 @@ def fetch_admin( bbox=None, admin_level=None, osm_level=None,
 
 
     """
+    if type(admin_level) is str:
+        admin_level = int(admin_level)
+    if type(osm_level) is str:
+        osm_level = int(osm_level)
 
     west, south, east, north = bbox
     assert admin_level in ADMIN_LEVELS, f'Invalid admin level. Valid values are {list(ADMIN_LEVELS.keys())}'

